components/namechangers.py

```diff
@@ -45,5 +45,5 @@ def compute_namechangers(df, options=None):
 
 def get_namechangers():
     options = Options(links_toggle=False, default_graph=Graph.last_16.value, average_foreground=True)
-    df = load_tourney_results(league_to_folder[champ])
+    df = load_tourney_results(league_to_folder[legend])
```

Comment:
One more change that basically invalidates the whole reason for a page, which is to track historical trends ;) Same as others, will lock it to champ until october 2024 unless I have time and will to work on porting this to both champ and legend